tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml.cs

```diff
@@ -14,17 +14,6 @@ public sealed partial class SummaryView : UserControl
     public SummaryView()
     {
         this.InitializeComponent();
-
```

Comment:
looks like Amir added this section last week. @AmelBawa-msft I think this part is still needed right?

Reply:
Spoke offline with Amir.  He agreed to remove this because he mis-read the figma.  Every task group should have a line above it.  and no group should have a line under it.  The first picture shows the correct border layout.